### PR TITLE
chore: update definition page

### LINKED ISSUE #889: [FEATURE] Ajouter les liens interne dans les pages de définition

### PR DESCRIPTION
**Pull request by johangirod**
ping @Jonathan2051 

fix #889

### CODE REVIEW

File: data/definitions/label-rge-reconnu-garant-de-l-environnement.yml
Comment (XavierJp) [inline data/definitions/label-rge-reconnu-garant-de-l-environnement.yml:1]: File name and topic dont match !

**Review by XavierJp — CHANGES_REQUESTED** [review]
Appart from `data/definitions/label-rge-reconnu-garant-de-l-environnement.yml`, sounds good to me

**Review by XavierJp — APPROVED** [review]

File: data/definitions/label-rge-reconnu-garant-de-l-environnement.yml
Comment (XavierJp) [reply data/definitions/label-rge-reconnu-garant-de-l-environnement.yml:1]: Fixed it and now deploying

Comment (johangirod) [reply data/definitions/label-rge-reconnu-garant-de-l-environnement.yml:1]: Thanks !

**Pull request merged**